__tests__/src/hooks/before.hook.js

```diff
@@ -0,0 +1,15 @@
+import { ApiClient } from "../apiClient.js";
+import { user } from "../data/user.js";
+
+export const mochaHooks = {
+  async beforeAll() {
+    const apiClient = ApiClient.unauthorized();
+    const { body } = await apiClient.user.register(user);
+    if (body.message.includes("Invalid login: 534-5.7.14")) {
```

Comment:
?

Reply:
will be removed, was for testing purposes =D